src/posix.mak

```diff
@@ -351,7 +351,7 @@ $G/backend.a: $(G_OBJS)
 $G/dmd_frontend: $(FRONT_SRCS) $D/gluelayer.d $(ROOT_SRCS) $G/newdelete.o $(STRING_IMPORT_FILES) $(HOST_DMD_PATH)
 	CC=$(HOST_CXX) $(HOST_DMD_RUN) -of$@ $(MODEL_FLAG) -vtls -J$G -J../res -L-lstdc++ $(DFLAGS) $(filter-out $(STRING_IMPORT_FILES) $(HOST_DMD_PATH),$^) -version=NoBackend
 
-dmd: $G/dmd
+dmd: $G/dmd $G/dmd.conf
```

Comment:
Yes it should, but it also needs to be copied to src/dmd.conf while we're still using that.
Also the target for `make -C src -f posix.mak dmd.conf` (src/posix.mak) is used in many places.